WORKSPACE

```diff
@@ -40,41 +47,79 @@ maybe(
 )
 
 load("@io_bazel_rules_scala//:scala_config.bzl", "scala_config")
+
 scala_config(scala_version = scala_version)
 
 load("@io_bazel_rules_scala//scala_proto:scala_proto.bzl", "scala_proto_repositories")
 load("@io_bazel_rules_scala//scala_proto:toolchains.bzl", "scala_proto_register_toolchains")
+
 scala_proto_repositories()
+
 scala_proto_register_toolchains()
 
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repositories")
-scala_repositories(fetch_sources = True)
+
+scala_repositories(
+    fetch_sources = True,
+)
 
 load("@io_bazel_rules_scala//scala:toolchains.bzl", "scala_register_toolchains")
+
 scala_register_toolchains()
 
 load("@io_bazel_rules_scala//testing:scalatest.bzl", "scalatest_repositories", "scalatest_toolchain")
+
 scalatest_repositories()
+
 scalatest_toolchain()
 
-# Bazel-Deps
-maybe(
-    http_file,
-    name = "bazeltools_bazel_deps_macos",
-    sha256 = "a37ebb76fefc3961c5c1cf5461395dcd18694f200ea12ea140947838d1946d19",
-    url = "https://github.com/bazeltools/bazel-deps/releases/download/v{}/bazel-deps-macos".format(bazel_deps_version),
-    executable = True,
+# rules_jvm_external
+
+http_archive(
+    name = "rules_jvm_external",
+    sha256 = "b17d7388feb9bfa7f2fa09031b32707df529f26c91ab9e5d909eb1676badd9a6",
+    strip_prefix = "rules_jvm_external-%s" % rules_jvm_external_version,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % rules_jvm_external_version,
 )
-maybe(
-    http_file,
-    name = "bazeltools_bazel_deps_linux",
-    sha256 = "bf9395f2d664cb4871fc48a1062e0b2e7d6cf0c0e442e115f654d275c5e6560e",
-    url = "https://github.com/bazeltools/bazel-deps/releases/download/v{}/bazel-deps-linux".format(bazel_deps_version),
-    executable = True,
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+
+rules_jvm_external_deps()
+
+load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
+
+rules_jvm_external_setup()
+
+# deps - TODO: how to put this section into a separate file? do we need a repository_rule for that?
```

Comment:
Yeah, any .bzl file could contain the maven_install invocation, and then be loaded in here.

Reply:
The difficulty was with the fact that after it there's a `load` from it and invoking the pinning check.